install/installer/traf_start

```diff
@@ -58,19 +58,19 @@ else
        error1392=$(grep "1392" $SQ_ROOT/initializeErrors.txt | wc -l)
        error1395=$(grep "1395" $SQ_ROOT/initializeErrors.txt | wc -l)
        if [ "$error1395" -ge "1" ] || [ "$error1392" -ge "1" ]; then
-          ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"get version of metadata;\" | sqlci" | tee -a $SQ_ROOT/initalizeErrors.txt
+          ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"get version of metadata;\" | sqlci" | tee -a $SQ_ROOT/initializeErrors.txt
            metadataCurrent=$(grep "Metadata is current" $SQ_ROOT/initializeErrors.txt | wc -l)
            if [[ "$metadataCurrent" -ne "1" ]]; then
               ssh $STARTING_NODE "cd $SQ_ROOT/sql/scripts;  echo \"initialize Trafodion, upgrade;\" | sqlci" | tee -a $SQ_ROOT/initializeErrors.txt
            fi
-        fi
-
-        if [ "$?" != "0" ]; then
-           echo "***ERROR: initialize Trafodion failed.  Check install log files for details." | tee -a $INSTALL_LOG
-           exit -1;
-        fi
+       else
```

Comment:
What if the initialize trafodion, upgrade fails, shouldn't we check that for errors as well? This else would prevent that as far as I can see. Maybe better to remove the "-a" from the tee command on line 64 and overwrite file initializeErrors.txt. Then remove this else statement on line 66 and continue unconditionally?